dashboard/app/models/levels/gamelab_jr.rb

```diff
@@ -25,8 +25,6 @@
 
 class GamelabJr < Gamelab
   serialized_attrs %w(
-    custom_helper_library
```

Comment:
What happens when we load a level that has an old `custom_helper_library` or `custom_blocks`?

Reply:
The data will still be there in the `properties` hash, but there won't be a getter or setter method for it.